app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep1Fragment.java

```diff
@@ -53,21 +60,39 @@
 
     private static final String ARG_SELECTED_CHAPTER = "selected_chapter";
     @Bind(R.id.chapter_spinner)
-    Spinner mChapterSpinner;
+    AutoCompleteSpinnerView autoCompleteSpinnerView;
     @Bind(R.id.confirm)
-    Button mConfirm;
+    Button mConfirmButton;
     @Bind(R.id.viewSwitcher)
     ViewSwitcher mLoadSwitcher;
-    private ChapterAdapter mSpinnerAdapter;
+    private ChapterAdapter mChapterForCityName;
```

Comment:
I don't think we need a separate dialog here. The list in autocomplete is I think enough. 
We can pre-fill it with a closest chapter. When user erases the chapter name, autocomplete can show all the chapters sorted by location.